src/s3.js

```diff
@@ -42,15 +42,15 @@ export async function* listObjects({bucket, prefix}) {
  * Generate a signed URL for an S3 object.
  *
  * @param {object} params
- * @param {string} params.method - HEAD, GET, PUT, DELETE
  * @param {string} params.bucket - name of bucket
  * @param {string} params.key - object key
+ * @param {string} [params.method] - HEAD, GET, PUT, DELETE. Default: GET
  * @param {object} [params.headers] - Content-Type and/or Content-Encoding headers for PUT
  * @param {number} [params.expiresIn] - seconds until the URL expires
  * @param {number} [params.issuedAt] - absolute time in seconds since the Unix epoch at which the signed URL should be considered issued at, i.e. when the countdown for expiresIn starts
  * @returns {string} signed URL
  */
-export async function signedUrl({method, bucket, key, headers, expiresIn, issuedAt} = {method: "GET", headers: {}}) {
+export async function signedUrl({method="GET", bucket, key, headers={}, expiresIn, issuedAt}) {
```

Comment:
FWIW, this dropped the trailing `= {}` that I'd intentionally suggested in `{method="GET", …} = {}`. Without it, a call to `signedUrl()` generates an error like:

    Uncaught TypeError: Cannot read properties of undefined (reading 'method')

While calling `signedUrl()` is always an error, omitting the default `= {}` seems a weirder way to throw the error compared to `signedUrl({})`:

    Uncaught Error: No value provided for input HTTP label: Bucket.